examples/client/get_event_messages.c

```diff
@@ -184,7 +198,7 @@ void process_event_data(event_client_event_t *event) {
            strstr(event->topic, "messages/tagged-data/") != NULL) {
     print_serialized_data(event->data, event->data_len);
   }
-  // check for topic messages/{messageId}/metadata
+  // check for topic message-metadata/{messageId}
   else if ((strstr(event->topic, "messages/") != NULL) && (strstr(event->topic, "/metadata") != NULL)) {
```

Comment:
if condition needs an update as well.
```suggestion
  // check for topic message-metadata/{messageId}
  else if (strcmp(event->topic, "messages-metadata/") != NULL ) {
```

Reply:
I missed that. Thank You.